migrations/20180625135525_add_treasure_status_to_stored_genesis_hashes.up.fizz

```diff
@@ -0,0 +1 @@
+add_column("stored_genesis_hashes", "treasure_status", "integer", {})
```

Comment:
Wouldn't this need it's own `down` migration file too?

Reply:
I think you're right.  I can add that in my next PR.